static/app/views/performance/newTraceDetails/traceApi/useTrace.tsx

```diff
@@ -78,7 +80,14 @@ export function getTraceQueryParams(
     delete otherParams.statsPeriod;
   }
 
-  const queryParams = {...otherParams, limit, timestamp, eventId, useSpans: 1};
+  const queryParams = {
```

Comment:
does this get forwarded to the trace endpoint?

Reply:
It doesn't hit the trace endpoint at all when there is a sampleSlug